src/coreComponents/physicsSolvers/multiphysics/CoupledSolver.hpp

```diff
@@ -84,13 +107,17 @@ class CoupledSolver : public PhysicsSolverBase
       using SolverPtr = TYPEOFREF( solver );
       using SolverType = TYPEOFPTR( SolverPtr {} );
       auto const & solverName = m_names[idx()];
-      auto const & solverType = LvArray::system::demangleType< SolverType >();
       solver = this->getParent().template getGroupPointer< SolverType >( solverName );
-      GEOS_THROW_IF( solver == nullptr,
-                     GEOS_FMT( "{}: Could not find solver '{}' of type {}",
-                               getDataContext(),
-                               solverName, solverType ),
-                     InputError );
+      if( solver== nullptr )
+      {
+        std::ostringstream errorMessage;
+        errorMessage << GEOS_FMT( "{}: Could not find solver named '{}'.\n",
+                                  getDataContext(), solverName );
```

Comment:
To target the solver attribute (xml line):
```suggestion
                                  getWrapperDataContext( SolverType::coupledSolverAttributePrefix() ),
                                  solverName );
```
then you can remove the `string const & solverType` parameter from `throwSolversNotFound()`